src/darwin/Framework/CHIP/MTRDeviceControllerDelegate.h

```diff
@@ -64,8 +66,8 @@ typedef NS_ENUM(NSUInteger, MTRPairingStatus) {
     MTRPairingStatusFailed API_DEPRECATED(
         "Please use MTRCommissioningStatusFailed", ios(16.1, 16.4), macos(13.0, 13.3), watchos(9.1, 9.4), tvos(16.1, 16.4))
     = 2,
-    MTRPairingStatusDiscoveringMoreDevices API_DEPRECATED("Please use MTRCommissioningStatusDiscoveringMoreDevices",
-        ios(16.1, 16.4), macos(13.0, 13.3), watchos(9.1, 9.4), tvos(16.1, 16.4))
+    MTRPairingStatusDiscoveringMoreDevices API_DEPRECATED("MTRPairingStatusDiscoveringMoreDevices is not used.", ios(16.1, 16.4),
```

Comment:
The second version in all these deprecation messages should be whatever version this will actually be released in.  The right thing to do is probably to add API_NEWLY_DEPRECATED to these and once we know the version numbers involved we can make these say the right things.